build_scripts/build_usd.py

```diff
@@ -539,8 +539,15 @@ def DownloadURL(url, context, force, extractDir = None,
     Returns the absolute path to the directory where files have 
     been extracted."""
     with CurrentWorkingDirectory(context.srcDir):
-        # Extract filename from URL and see if file already exists. 
-        filename = url.split("/")[-1]       
+        # Extract filename from URL.
+        filename = url.split("/")[-1]
+
+        # Change filename for a file form a github/gitlab repository. 
+        # Filename becomes projectName-version to avoid file collision.
+        if ("github" or "gitlab") in url:
```

Comment:
The expression `"github" or "gitlab"` will always evaluate to `"github"` so it's not actually checking for the presence of `gitlab` in the URL.

The only usage of `gitlab` in `build_usd.py` points to a named release as well, so you can probably just drop the check for `gitlab`.
```
> grep -r gitlab build_scripts/
build_scripts/build_usd.py:TIFF_URL = "https://gitlab.com/libtiff/libtiff/-/archive/v4.0.7/libtiff-v4.0.7.zip"
```

Reply:
Thank you for pointing this out. I checked the GitLab too quickly, since the output name contains the name of the project.

Will replace with an `if any(...)` still want to check against GitLab URL just in case.